libevm/VMOpt.cpp

```diff
@@ -63,14 +63,16 @@ void VM::copyCode(int _extraBytes)
 
 void VM::optimize()
 {
+	copyCode(33);
```

Comment:
It won't need more than 33 extra bytes ever with the current optimizations.  But it needs them here, not later, thanks.